app/script/localization/translations/webapp-de.js

```diff
@@ -274,6 +275,9 @@ z.string.de.modalUserUnblockAction = 'Freigeben';
 z.string.de.modalUserUnblockHeadline = 'Freigeben?';
 z.string.de.modalUserUnblockMessage = '{{user}} wird dich wieder kontaktieren und zu Gruppen hinzufügen können.';
 
+z.string.de.modalNoCameraTitle = 'Kein Kamerazugriff';
+z.string.de.modalNoCameraMessage = 'Wire hat keinen Zugriff auf die Kamera.[br][faqLink]Zur Behebung bitte diesen Support Artikel anschauen[/faqLink].';
+
```

Comment:
@infotexture Also somehwat weird in my opinion.

Reply:
Gregor, thanks for the feedback. As discussed on Friday in German localization chat: It is not ideal,  I will look into it.